sdk/program/src/vote/state/mod.rs

```diff
@@ -221,16 +221,21 @@ impl VoteStateUpdate {
 
 #[cfg_attr(
     feature = "frozen-abi",
-    frozen_abi(digest = "5PFw9pyF1UG1DXVsw7gpjHegNyRycAAxWf2GA9wUXPs5"),
+    frozen_abi(digest = "DqMoZ6hSsG1wguKx49kjZmU8sEu7jjzDWqg14qxoeinM"),
     derive(AbiExample)
 )]
 #[derive(Serialize, Default, Deserialize, Debug, PartialEq, Eq, Clone)]
 pub struct TowerSync {
-    /// The proposed tower
+    /// the proposed tower
     pub lockouts: VecDeque<Lockout>,
-    /// The proposed root
+    /// the proposed root
     pub root: Option<Slot>,
-    /// signature of the bank's state at the last slot
+    /// signature of the VoteState's at the latest slot in `lockouts`
+    pub yolo_hash: Hash,
+    /// the latest fully replayed slot, equal to the last
+    /// slot in `lockouts` until asynchronous execution.
```

Comment:
Do you mean "until asynchronous execution happens?"

Reply:
ya will update the comment